<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasswordForm</class>
 <widget class="QWidget" name="PasswordForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>241</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="widget_5" native="true">
     <property name="styleSheet">
      <string notr="true">QPushButton#pushButton, #pushButton_6{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(11, 131, 120, 219), stop:1 rgba(85, 98, 112, 226));
	color:rgba(255, 255, 255, 210);
	border-radius:5px;
}

QPushButton#pushButton:hover, #pushButton_6:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(150, 123, 111, 219), stop:1 rgba(85, 81, 84, 226));
}

QPushButton#pushButton:pressed, #pushButton_6:pressed{
	padding-left:5px;
	padding-top:5px;
	background-color:rgba(150, 123, 111, 255);
}

QPushButton#pushButton_2, #pushButton_3, #pushButton_4, #pushButton_5{
	background-color: rgba(0, 0, 0, 0);
	color:rgba(85, 98, 112, 255);
}

QPushButton#pushButton_2:hover, #pushButton_3:hover, #pushButton_4:hover, #pushButton_5:hover{
	color: rgba(131, 96, 53, 255);
}

QPushButton#pushButton_2:pressed, #pushButton_3:pressed, #pushButton_4:pressed, #pushButton_5:pressed{
	padding-left:5px;
	padding-top:5px;
	color:rgba(91, 88, 53, 255);
}

</string>
     </property>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>351</width>
        <height>221</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgba(255, 255, 255, 255);
color:rgba(255,255,255,200);
border-radius:5px;
border-top-left-radius: 50px;
border-bottom-right-radius: 50px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="widget_6" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>351</width>
        <height>221</height>
       </rect>
      </property>
      <widget class="QLineEdit" name="lineEdit_yenisifre">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgba(0, 0, 0, 0);
border:none;
border-bottom:2px solid rgba(46, 82, 101, 200);
color:rgba(0, 0, 0, 240);
padding-bottom:7px;</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>  Yeni Şifre</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_yenisifretekrar">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>160</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgba(0, 0, 0, 0);
border:none;
border-bottom:2px solid rgba(46, 82, 101, 200);
color:rgba(0, 0, 0, 240);
padding-bottom:7px;</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>  Yeni Şifre Tekrar</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_guvenlikkodu">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>110</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgba(0, 0, 0, 0);
border:none;
border-bottom:2px solid rgba(46, 82, 101, 200);
color:rgba(0, 0, 0, 240);
padding-bottom:7px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
       <property name="placeholderText">
        <string>  Güvenlik Kodu</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_onayla">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>160</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>8</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#pushButton_onayla {
    background-color: rgba(2, 65, 118, 255); /* Mavi arkaplan */
    color: white; /* Beyaz yazı rengi */
    border-radius: 5px;
}

QPushButton#pushButton_onayla:pressed {
    padding-left: 5px;
    padding-top: 5px;
    background-color: rgba(2, 65, 118, 100);
    background-position: calc(100% - 10px) center;
}

QPushButton#pushButton_onayla:hover {
    background-color: rgba(2, 65, 118, 150); /* İmleç üstüne gelindiğinde hafif ton farkı */
}</string>
       </property>
       <property name="text">
        <string>Onayla</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>351</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgba(0, 0, 0, 200);</string>
       </property>
       <property name="text">
        <string>Şifrenizi Belirleyin</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_eposta3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>59</y>
         <width>291</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgba(0, 0, 0, 0);
border:none;
border-bottom:2px solid rgba(46, 82, 101, 200);
color:rgba(0, 0, 0, 240);
padding-bottom:7px;</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
       <property name="placeholderText">
        <string>  E-posta</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
